RUFAS/routines/field/field/field.py

```diff
@@ -688,10 +701,25 @@ def _record_manure_application(
             Mass of potassium in the manure applied (kg)
 
         """
+        units = {
+            "dry_matter_mass": "kg",
+            "dry_matter_fraction": "unitless",
+            "field_coverage": "unitless",
+            "application_depth": "mm",
+            "surface_remainder_fraction": "unitless",
+            "nitrogen": "kg",
+            "phosphorus": "kg",
+            "potassium": "kg",
+            "day": "unitless",
+            "year": "unitless",
+            "field_size": "ha",
+            "average_clay_percent": "unitless",
```

Comment:
```suggestion
            "dry_matter_mass": "dry kg",
            "dry_matter_fraction": "fraction",
            "field_coverage": "unitless",
            "application_depth": "mm",
            "surface_remainder_fraction": "unitless",
            "nitrogen": "kg",
            "phosphorus": "kg",
            "potassium": "kg",
            "day": "day",
            "year": "year",
            "field_size": "ha",
            "average_clay_percent": "percent",
```
Same uncertainties about percents vs fractions and explicitness as above.